catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceCallable.java

```diff
@@ -68,7 +64,7 @@
     private boolean cancelDownload = false;
 
     /**
-     * Used when only downloading, no caching to @FileOutputStream because caching was disabled or 
+     * Used when only downloading, no caching to @FileOutputStream because caching was disabled or
```

Comment:
These `@ClassName` are all over this class. We should replace all of them with `{@link ClassName}`.